chart/epinio/templates/stage-scripts.yaml

```diff
@@ -35,7 +36,7 @@ data:
     # command is not available in the `bash` image. The code as written now relies on each
     # unpacker to recognize/reject input properly.
     #
-    # The `sed` command add format indicators to the output from the unarchivers, making
+    # The `sed` commands add format indicators to the output from the unarchivers, making
```

Comment:
```suggestion
    # The `sed` command adds format indicators to the output from the unarchivers, making
```

Reply:
I put the plural on the command because there are several `sed`s, one per possible archive format.